department-of-veterans-affairs/abd-vro
# use an existing role for postgres health check

## What was the problem?


health check (pg_isready) in docker-compose for postgres was using the default role (postgres) .

Associated tickets or Slack threads:

- [MCP-000](https://amida.atlassian.net/browse/MCP-000)

## How does this fix it?[^1]

adds options so that pg_isready uses existing role

## How to test this PR
- Step 1: Build application locally
- Step 2: Take a look at postgres service container log to make sure no error message is. coming every 10 seconds


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
